src/main/groovy/wooga/gradle/unity/UnityPluginExtension.groovy

```diff
@@ -24,14 +24,14 @@ import wooga.gradle.unity.batchMode.*
 
 interface UnityPluginExtension extends UnityPluginTestExtension {
 
-    File getUnityPath()
-
     void setUnityPath(Object path)
 
-    File getUnityLicenseDirectory()
```

Comment:
Why are these properties flipped?

Reply:
i think it's ok since we don't provide a setter